data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json

```
@@ -73,5 +73,180 @@
     "symbol": "O",
     "color": "white_magenta",
     "use_action": { "type": "learn_spell", "spells": [ "self_healing" ] }
+  },
+  {
+    "id": "blindfold_artifact",
+    "type": "ARMOR",
+    "copy-from": "blindfold",
+    "looks_like": "blindfold",
+    "name": { "str": "red devil blindfold" },
+    "description": "A red fabric blindfold with a cartoonish devil over the eyes covering tied over the eyes to block sight.  You feel some anger when you put it on.  Did that devil just wink at you?",
```

```suggestion
    "description": "A red fabric blindfold with a cartoonish devil over the eyes.  You feel some anger when you put it on.  Did that devil just wink at you?",
```
It's a blindfold, function is assumed :p